releasenotes/notes/upgrade-coherence-limit-9a1fccac402b4f5c.yaml

```diff
@@ -0,0 +1,8 @@
+---
+upgrade:
+  - |
+    A function computing the coherence limit error is renamed to
```

Comment:
I think most impactful change to users is now it can take any number of qubits.

Reply:
Update in 8ff0c20 is close to what you expected?

Reply:
Thanks, looks good.